<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logbook::Internal::CallsignLookupSettingsPage</class>
 <widget class="QWidget" name="Logbook::Internal::CallsignLookupSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>QRZ.com</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edQRZPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
         <property name="buddy">
          <cstring>edQRZPassword</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edQRZUsername"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
         <property name="buddy">
          <cstring>edQRZUsername</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chkQRZEnabled">
         <property name="text">
          <string>Enabled</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="chkLookupEnabled">
     <property name="text">
      <string>Callsign Lookup Enabled</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkLookupEnabled</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>chkQRZEnabled</tabstop>
  <tabstop>edQRZUsername</tabstop>
  <tabstop>edQRZPassword</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkQRZEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>edQRZUsername</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkQRZEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>edQRZPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkLookupEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>tabWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
